modules/lvgl/lvgl_display_mono.c

```diff
@@ -9,6 +9,9 @@
 #include <string.h>
 #include "lvgl_display.h"
 
+#include <zephyr/logging/log.h>
+LOG_MODULE_REGISTER(lvgl_display_mono, CONFIG_DISPLAY_LOG_LEVEL);
```

Comment:
I would use one module across all `lvgl_display_*` to cut down on flash usage, since usually there is only one active at a time anyways.

Reply:
Add the `LOG_MODULE_REGISTER` to `lvgl_display.c` and in here use `LOG_MODULE_DECLARE`